plugins/qlcplus_4.12.1/presets.js

```diff
@@ -0,0 +1,1002 @@
+  isDecreasingSpeed: cap => cap.speed !== null && Math.abs(cap.speed[0].number) > Math.abs(cap.speed[1].number),
+  isStopped: cap => cap.speed !== null && cap.speed[0].number === 0 && cap.speed[1].number === 0,
+  isIncreasingDuration: cap => cap.duration !== null && Math.abs(cap.duration[0].number) < Math.abs(cap.duration[1].number),
+  isDecreasingDuration: cap => cap.duration !== null && Math.abs(cap.duration[0].number) > Math.abs(cap.duration[1].number),
+  isColorIntensity: (cap, color) => cap.type === `ColorIntensity` && cap.color === color,
+  isShutterEffect: (cap, shutterEffect) => cap.type === `ShutterStrobe` && cap.shutterEffect === shutterEffect,
+  hasFrequency: cap => cap.speed !== null && (cap.speed[0].unit === `Hz` || cap.speed[0].unit === `bpm`),
+  isRotationSpeed: cap => (cap.type.endsWith(`Rotation`) || [`PanContinuous`, `TiltContinuous`, `Prism`].includes(cap.type)) && cap.speed !== null,
+  isRotationAngle: cap => (cap.type.endsWith(`Rotation`) || [`Pan`, `Tilt`, `Prism`].includes(cap.type)) && cap.angle !== null,
+  isBeamAngle: cap => (cap.type === `BeamAngle` || cap.type === `Zoom`) && cap.angle !== null,
+  isWheelChannel: channel => channel.capabilities.some(cap => [`WheelSlot`, `WheelRotation`].includes(cap.type)),
+  isAllowedInWheels: cap => [`WheelSlot`, `WheelShake`, `WheelSlotRotation`, `WheelRotation`, `Effect`, `NoFunction`].includes(cap.type),
+
+  /**
+   * @param {Entity} entity The speed Entity object.
+   * @returns {number|null} The frequency in Hertz, or null, if the entity's unit is not convertable to Hertz.
+   */
+  getFrequencyInHertz(entity) {
+    if (entity.unit === `Hz`) {
+      return entity.number;
+    }
+
+    if (entity.unit === `bpm`) {
+      return entity.number / 60;
+    }
+
+    return null;
+  }
+};
+
+const importHelpers = {
+  getColorIntensityCap: color => ({
+    type: `ColorIntensity`,
+    color
+  }),
+
+  getPanTiltCap: (panOrTilt, maxValue) => {
+    const cap = {
+      type: panOrTilt,
+      angleStart: `0deg`,
+      angleEnd: `${maxValue}deg`
+    };
+
+    if (maxValue === 0) {
+      cap.angleStart = `0%`;
+      cap.angleEnd = `100%`;
+      cap.helpWanted = `Can you provide exact angles?`;
+    }
+
+    return cap;
+  },
+
+  getShutterStrobeCap: (shutterEffect, speedStart, speedEnd, randomTiming) => {
+    const cap = {
+      type: `ShutterStrobe`,
+      shutterEffect
+    };
+
+    if (speedEnd) {
+      cap.speedStart = speedStart;
+      cap.speedEnd = speedEnd;
+    }
+    else if (speedStart) {
+      cap.speed = speedStart;
+    }
+
+    if (randomTiming) {
+      cap.randomTiming = true;
+    }
+
+    return cap;
+  },
+
+  getRotationCapType: ({ channelName, channelType }) => {
+    if ([`Pan`, `Tilt`].includes(channelType)) {
+      return `${channelType}Continuous`;
+    }
+
+    if ([`Colour`, `Gobo`].includes(channelType) || /\bgobo\b/i.test(channelName)) {
+      return /wheel\b/i.test(channelName) ? `WheelRotation` : `WheelSlotRotation`;
```

Comment:
I won't use `\b` in both Regexes here. There will certainly be channels called `Gobowheel`, and `Wheelrotation` might also be written as one word. Anyway, which false positives to you want to sort out by using `\b`?